source/_components/device_tracker.traccar.markdown

```diff
@@ -60,4 +60,13 @@ verify_ssl:
   required: false
   type: boolean
   default: true
-{% endconfiguration %}
\ No newline at end of file
+scan_interval:
+  description: Defines the polling interval.
+  required: false
+  type: time
+  default: 00:00:30
+monitored_conditions:
+  description: Additional traccar computed attributes or device-related attributes to include in the scan.
+  required: false
+  type: list
```

Comment:
Where are the keys?

Reply:
I can't define keys, the attributes are custom defined in the traccar platform, they can be any name the user wants.

Reply:
I think we should include this information, normally you have a list that you can choose from. This can otherwise create confusion.

Reply:
Where do you think this can be clarified? In the parameter description?

Reply:
My advice is to make a separate paragraph for it, with how the `monitored_conditions` are proportionate to what you set up in traccar.